srcs/client/src/Components/Tools/NavBar/NavBar.css

```diff
@@ -1,37 +1,196 @@
 
-.navBar{
-	height: 8vw;
-	width: auto;
-	display: flex;
-	flex-direction: row;
-	/* border: 5px solid white; */
-	align-items: center;
-    /* background-color: transparent; */
+/* .navbar-toggler {
+	border: 3px solid #80FFFF;
+	box-shadow: 0px 4px 15px #80FFFF,
+				4px 0px 10px rgba(30, 166, 166, 0.95),
+				0px 0px 10px rgba(255, 255, 255, 0.521);
+} */
+.navbar-toggler-icon {
+	border: 5px solid #80FFFF;
+	margin-top:4px;
+	
+	margin-bottom:5px;
+	border-radius: 25px;
+	box-shadow: 0px 1px 15px #80FFFF,
+				3px 0px 8px rgba(30, 166, 166, 0.95),
+				0px 0px 8px rgba(255, 255, 255, 0.521);
+	background-image: url();
+	/* background-image: url(data:image/svg+xml,%3csvg xmlns=%27http://www.w3.org/2000/svg%27 viewBox=%270 0 30 30%27%3e%3cpath stroke=%27rgba%284, 247, 249, 0.55%29%27 stroke-linecap=%27round%27 stroke-miterlimit=%2710%27 stroke-width=%272%27 d=%27M4 7h22M4 15h22M4 23h22%27/%3e%3c/svg%3e); */
+}
+
+.nav-link {
+	margin-left:8px;
+	margin-right:8px;
+  }
+
+  a {
+	text-decoration: none; 
+
+	/* text decoration  */
+	position:relative;
+	padding: 10px 0px;
+	color: #FFE710;
+	text-transform: uppercase;
+	letter-spacing: 2px;
+	font-size: 24px;
```

Comment:
I am concerned by this regarding reusability : as we are using standard html elements (<a> here), we can easily apply these styles by mistake on another context on another page and it will result with having a huge text instead of just a link.

Reply:
I think we don't have to be afraid of this technique (using the generic class) because navbar.css is only use in Navbar.tsx . We will have a more readable Jsx code with this, it will save a lot div /line (around 16 ). I'm open to discuss this point ;)